deploy/managed-upgrade-operator-config/4.7/10-managed-upgrade-operator-configmap.yaml

```diff
@@ -52,6 +52,7 @@ data:
       - ThanosRuleSenderIsFailingAlerts
       - ThanosRuleHighRuleEvaluationFailures
       - ThanosNoRuleEvaluations
+      - etcdGRPCRequestsSlow
```

Comment:
When adding an alert to this list, please add a leading comment line (`# ....`) that links to the Jira ticket where this decision was made.

Reply:
Added change, Thanks :) @mrbarge